setup.py

```diff
@@ -8,7 +8,7 @@
 setup(
     name='banana_dev',
     packages=['banana_dev'],
-    version='5.0.2',
+    version='5.0.3',
```

Comment:
nit: shouldn't this be a non breaking feature change? ie, 5.1.0?

Reply:
sure let's start doing that